### DISCUSSION TITLE
Can an individual user install an extension?

### DISCUSSION THREAD

**Discussion by pitosalas** [coder/code-server#2806] [General] [answered]
When I try I get this error:

<img width="556" alt="Screen Shot 2021-03-02 at 10 17 04 AM" src="https://user-images.githubusercontent.com/44681/109672187-c6e6bb00-7b42-11eb-8af3-82ddc1093126.png">

For example, the ms python extension, seems not to do anything. In particular no "python" commands appear. Am I doing something wrong?

**Comment by code-asher** [comment, marked as answer]
Our version of the Python extension is currently broken (well technically the Jupyter extension it depends on is broken). Some workarounds here: https://github.com/cdr/code-server/issues/2341

I'm not sure if the extension requires service workers, if it does you'll probably need to use HTTPS to enable service workers.
